### ISSUE TITLE
Layotter breaks with ACF 5.7.11

### ISSUE THREAD

**Issue by MatzeKitt** [hingst/layotter#33]
Currently, Layotter doesn’t work with ACF 5.7.11, which has been released yesterday. Every frontend is broken with the following error message:

```
Fatal error: Uncaught Error: Call to undefined function _acf_get_field_group_by_key() in /var/www/wd50/wp-content/plugins/layotter/core/acf-abstraction.php on line 237
( ! ) Error: Call to undefined function _acf_get_field_group_by_key() in /var/www/wd50/wp-content/plugins/layotter/core/acf-abstraction.php on line 237
Call Stack
#	Time	Memory	Function	Location
1	0.0001	361656	{main}( )	.../index.php:0
2	0.0002	361944	require( '/var/www/wd50/wp-blog-header.php' )	.../index.php:17
3	0.0002	362296	require_once( '/var/www/wd50/wp-load.php' )	.../wp-blog-header.php:13
4	0.0002	363032	require_once( '/var/www/wd50/wp-config.php' )	.../wp-load.php:37
5	0.0002	365512	require_once( '/var/www/wd50/wp-settings.php' )	.../wp-config.php:125
6	0.2017	9078688	do_action( )	.../wp-settings.php:467
7	0.2017	9079064	WP_Hook->do_action( )	.../plugin.php:453
8	0.2017	9079064	WP_Hook->apply_filters( )	.../class-wp-hook.php:310
9	0.2341	9346720	ACF->init( )	.../class-wp-hook.php:286
10	0.3058	12309976	do_action( )	.../acf.php:336
11	0.3058	12310352	WP_Hook->do_action( )	.../plugin.php:453
12	0.3058	12310352	WP_Hook->apply_filters( )	.../class-wp-hook.php:310
13	0.3058	12311480	LayotterWidgets->init( )	.../class-wp-hook.php:286
14	0.3060	12311480	Layotter::get_filtered_element_types( )	.../LayotterWidgets.php:51
15	0.3060	12312176	Layotter::create_element( )	.../core.php:105
16	0.3060	12312776	RH_Accordion->__construct( )	.../core.php:70
17	0.3060	12312776	RH_Accordion->get_fields( )	.../element.php:75
18	0.3060	12312776	Layotter_ACF::get_field_group_by_key( )	.../element.php:103
```

**Comment by maxleistner** [comment]
Seems like they have refactored some core functions:

```
5.7.11 Release Date - 11 Februrary 2019
New - Added support for persistent object caching.
Dev - Refactored various core functions.
Dev - Added new hook variation functions acf_add_filter_variations() and acf_add_action_variations().
```

**Comment by hingst** [comment]
Thanks for reporting, I'll have a look this evening

**Comment by jakesch** [comment]
Unfortunately I get a similar Error Message:

<pre><code>Fatal error: Uncaught Error: Call to undefined method ACF::get_info() in /app/public/wp-content/plugins/layotter-master/core/acf-abstraction.php on line 44
Error: Call to undefined method ACF::get_info() in /app/public/wp-content/plugins/layotter-master/core/acf-abstraction.php on line 44
Call Stack
#	Time	Memory	Function	Location
1	0.0011	365352	{main}( )	.../plugins.php:0
2	0.0016	366024	require_once( '/app/public/wp-admin/admin.php' )	.../plugins.php:10
3	0.0020	366504	require_once( '/app/public/wp-load.php' )	.../admin.php:31
4	0.0027	367280	require_once( '/app/public/wp-config.php' )	.../wp-load.php:37
5	0.0031	374248	require_once( '/app/public/wp-settings.php' )	.../wp-config.php:81
6	0.2844	2973384	do_action( )	.../wp-settings.php:327
7	0.2844	2973760	WP_Hook->do_action( )	.../plugin.php:453
8	0.2844	2973760	WP_Hook->apply_filters( )	.../class-wp-hook.php:310
9	0.3210	3506000	layotter( )	.../class-wp-hook.php:286
10	0.3269	3574592	Layotter_ACF::is_available( )	.../index.php:30
11	0.3269	3574592	Layotter_ACF::is_version_compatible( )	.../acf-abstraction.php:57
</code></pre>

I tried several Combinations with Layotter and ACF Free Versions, but it was everytime the same report. Now I have installed the latests Version of both of them (1.5.26 and 5.7.12)

Now the interesting Part: with the latest Version of ACF Pro the Error doesn't occur.

**Comment by hingst** [comment]
Hi @jakesch, I only just now stumbled upon your comment, sorry. I just created #35 and will fix that asap.

**Comment by hingst** [comment]
@jakesch fixed in v1.5.27
